bin/run_verify_reqs.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Verifies that the requirements file is built by the version of Python that
+# runs in the container.
+
+cp requirements.txt requirements.txt.orig
+pip-compile --quiet --generate-hashes
+diff requirements.txt requirements.txt.orig
```

Comment:
This uses `diff` rather than `git diff` because `.git` isn't in the container. I wanted to run this script in the container because then it guarantees it's using the right version of Python and I don't have to deal with what I had before where we had multiple container specifications and I'd forget to update them all at the same time and then "strange things occur".